monkestation/code/game/machinery/exp_cloner.dm

```diff
@@ -54,13 +73,20 @@
 	ADD_TRAIT(clonee, TRAIT_NOCRITDAMAGE, CLONING_POD_TRAIT)
 	clonee.Unconscious(80)
 
+	if(!isnull(evil_objective))
+		role_text = "evil clone"
```

Comment:
since these vars are always set here they can just be on the proc instead of the object itself

Reply:
What do you mean by this?

Reply:
Ok I hope I did the right change

Reply:
yes this is correct

Reply:
Horay